.cirrus.yml

```diff
@@ -1,5 +1,19 @@
 gcp_credentials: ENCRYPTED[1816835da1e936dabb469b51501856ec8468676e35e967bd0fd720a815498e5ee6c8a6a79219ce273f67bcb8f1aa948a]
 
+tool_setup_template: &TOOL_SETUP_TEMPLATE
+  tool_setup_script:
+    - git fetch origin master # To set FETCH_HEAD for "git merge-base" to work
+    # Pinned version of the plugin tools, to avoid breakage in this repository
+    # when pushing updates from flutter/plugins.
+    - pub global activate flutter_plugin_tools 0.0.45+3
```

Comment:
I didn't get why we need to pin a specific version. Can't the package repo use the new version that to be published? Or you plan to update it to latest version in a separate PR?

Reply:
> Can't the package repo use the new version that to be published? Or you plan to update it to latest version in a separate PR?

Yes and yes. The issue is that we don't have any real concept of "breaking changes" to the tool, at least in the new structure (not sure how it worked before). When we land things we'll only know they work in flutter/plugins, not that they don't break flutter/packages. By pinning the version and then explicitly rolling here, we'll avoid surprise breakage to flutter/plugins from some non-obvious source.